### PR TITLE
dm: enhance description about error triggering condition

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Enhance description about error triggering condition

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by D3Hunter — COMMENTED** [review]
lgtm. @lance6716 ptal

**Review by D3Hunter — APPROVED** [review]

File: dm/dm-handle-alerts.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
 
 ### `DM_relay_process_exits_with_error`
 
-当 relay log 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
+当 relay log 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```
Comment (lance6716) [inline dm/dm-handle-alerts.md:39]:
```suggestion
当 relay log 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```

Comment (lance6716) [reply dm/dm-handle-alerts.md:39]: 在遇到可自动恢复错误时不会保持 Paused 状态，下同

File: dm/dm-handle-alerts.md [hunk 2]
```diff
@@ -66,17 +66,17 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
 
 ### `DM_dump_process_exists_with_error`
 
-当 Dump 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
+当 Dump 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```
Comment (hfxsd) [inline dm/dm-handle-alerts.md:69]:
```suggestion
当 Dump 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```

File: dm/dm-handle-alerts.md [hunk 3]
```diff
@@ -66,17 +66,17 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
...
 
 ### `DM_load_process_exists_with_error`
 
-当 Load 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
+当 Load 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```
Comment (hfxsd) [inline dm/dm-handle-alerts.md:73]:
```suggestion
当 Load 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```

File: dm/dm-handle-alerts.md [hunk 4]
```diff
@@ -66,17 +66,17 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
...
 
 ## Binlog replication 告警
 
 ### `DM_sync_process_exists_with_error`
 
-当 Binlog replication 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
+当 Binlog replication 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```
Comment (hfxsd) [inline dm/dm-handle-alerts.md:79]:
```suggestion
当 Binlog replication 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```

**Review by lance6716 — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: dm/dm-handle-alerts.md [hunk 5]
```diff
@@ -36,7 +36,7 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
 
 ### `DM_relay_process_exits_with_error`
 
-当 relay log 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
+当 relay log 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```
Comment (qiancai) [inline dm/dm-handle-alerts.md:39]:
```suggestion
当 relay log 处理单元遇到无法自动恢复的错误时（如找不到 binlog 文件），或者短时间内多次遇到可自动恢复的错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```

File: dm/dm-handle-alerts.md [hunk 6]
```diff
@@ -36,7 +36,7 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
...
```
Comment (qiancai) [inline dm/dm-handle-alerts.md:40]: 短时间是多短呢，可以说下具体的时间吗

Comment (D3Hunter) [reply dm/dm-handle-alerts.md:40]: 2分钟内遇到3 个以上的可自动恢复错误

File: dm/dm-handle-alerts.md [hunk 7]
```diff
@@ -36,7 +36,7 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
 
 ### `DM_relay_process_exits_with_error`
 
-当 relay log 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
+当 relay log 处理单元遇到无法自动恢复的错误时（如找不到 binlog 文件），或者短时间内多次遇到（如 2 分钟内遇到 3 次以上）自动恢复的错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```
Comment (qiancai) [inline dm/dm-handle-alerts.md:39]:
```suggestion
当 relay log 处理单元遇到无法自动恢复的错误时（如找不到 binlog 文件），或者短时间内多次遇到（如 2 分钟内遇到 3 次以上）可自动恢复的错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```

File: dm/dm-handle-alerts.md [hunk 8]
```diff
@@ -66,17 +66,17 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
 
 ### `DM_dump_process_exists_with_error`
 
-当 Dump 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
+当 Dump 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```
Comment (qiancai) [inline dm/dm-handle-alerts.md:69]:
```suggestion
当 Dump 处理单元遇到无法自动恢复的错误时（如找不到 binlog 文件），或者短时间内多次遇到（如 2 分钟内遇到 3 次以上）可自动恢复的错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```

File: dm/dm-handle-alerts.md [hunk 9]
```diff
@@ -66,17 +66,17 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
...
 
 ### `DM_load_process_exists_with_error`
 
-当 Load 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
+当 Load 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```
Comment (qiancai) [inline dm/dm-handle-alerts.md:73]:
```suggestion
当 Load 处理单元遇到无法自动恢复的错误时（如找不到 binlog 文件），或者短时间内多次遇到（如 2 分钟内遇到 3 次以上）可自动恢复的错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```

File: dm/dm-handle-alerts.md [hunk 10]
```diff
@@ -66,17 +66,17 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
...
 
 ## Binlog replication 告警
 
 ### `DM_sync_process_exists_with_error`
 
-当 Binlog replication 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
+当 Binlog replication 处理单元遇到不可自动恢复错误时（如找不到 binlog 文件），或者短时间内遇到多次可自动恢复错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```
Comment (qiancai) [inline dm/dm-handle-alerts.md:79]:
```suggestion
当 Binlog replication 处理单元遇到无法自动恢复的错误时（如找不到 binlog 文件），或者短时间内多次遇到（如 2 分钟内遇到 3 次以上）可自动恢复的错误时（如网络问题），会触发该告警，此时需要参考 [DM 故障诊断](/dm/dm-error-handling.md#dm-故障诊断)进行处理。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**